### PR TITLE
Update build_base images and related Dockerfile

### PR DESCRIPTION
**Pull request by iv0rish**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ ENV KLAYTN_DISABLE_SYMBOL=$KLAYTN_DISABLE_SYMBOL
 ADD . $SRC_DIR
```
Comment (aidan-kwon) [inline Dockerfile:20]: During the `make all` execution, `git status` is commanded. However, a container doesn't have `.git` file. 
```suggestion
RUN git init
ADD . $SRC_DIR
```

**Comment by aidan-kwon** [comment]
Ubuntu 20.04 uses GLIBC_2.29, and Ubuntu 18.04 doesn't support the glibc version. 
Please, give me one more day to upgrade the Ubuntu version we are using in Klaytn-deploy.

File: Dockerfile [hunk 2]
```diff
@@ -20,7 +20,7 @@ ENV KLAYTN_DISABLE_SYMBOL=$KLAYTN_DISABLE_SYMBOL
...
 RUN cd $SRC_DIR && make all
 
-FROM alpine:3
+FROM ubuntu:20.04
 ARG SRC_DIR
 ARG PKG_DIR
```
[the remark this answers is not in the document]
Comment (iv0rish) [reply Dockerfile:26]: I've applied your suggestion, thanks for the review.

**Review by aidan-kwon — APPROVED** [review]
Thank you

**Comment by aidan-kwon** [comment]
We need to update the build base of circle CI at the same time. 
https://github.com/klaytn/klaytn/blob/97b8e677ec1819668155315a43cfa623b1f03a19/.circleci/config.yml#L24-L29

**Comment by iv0rish** [comment]
@aidan-kwon I've updated CI pipeline to support new build_base and also remove some deprecated dependencies in CircleCI. All pipelines seems working fine but please review those if there's any problems behind.

**Review by aidan-kwon — APPROVED** [review]
The change looks good to me, but perhaps we need some more configuration during the deployment process. Let's test this with v1.8.3 release PR

**Review by kjhman21 — APPROVED** [review]
LGTM

**Pull request merged**